Applications/DataExplorer/VtkVis/VisualizationWidget.cpp

```diff
@@ -45,7 +45,7 @@
 #include <QString>
 
 VisualizationWidget::VisualizationWidget( QWidget* parent /*= 0*/ )
-	: QWidget(parent)
+	: _vtkRender(nullptr), _markerWidget(nullptr), QWidget(parent)
```

Comment:
warning: field '_markerWidget' will be initialized after base 'QWidget'
The order either in declarations or in the initialization list should be change and be the same.